react-jsx/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "react-jsx",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "webpack --watch --mode=development --devtool=source-map",
```

Comment:
The `build` and `watch` scripts should be 2 separate scripts. One is to be used while in development and the other to build the project for deployment.